examples/aura.yaml

```
@@ -0,0 +1,106 @@
+participants:
+  - el_client_type: nethermind
+    el_client_image: nethermind/nethermind:kurtosis
+    el_client_log_level: ""
+    el_extra_params: [
+      "--config=/nethermind/configs/validator.cfg",
+      "--Init.ChainSpecPath=/nethermind/chainspec/chainspec.json"
+    ]
+    cl_client_type: na
+    cl_client_image: null
+    cl_client_log_level: ""
+    beacon_extra_params: []
+    validator_extra_params: []
+    builder_network_params: null
+    validator_count: null
```

```suggestion
    validator_count: 0
```
If you set this to 0 it will not bother you with a validator process at all.

---

setting `validator_count: 0` for the first participant throws an error `failed to compute sync committee indices: no active validators to compute sync committee from`